### ISSUE TITLE
Add W014 and W015 to Bootlint wiki

### ISSUE THREAD

**Issue by kkirsche** [twbs/bootlint#256]
W014 — Lint Carousel Controls — https://github.com/twbs/bootlint/blob/master/src/bootlint.js#L979
W015 — Lint New Bootstrap — https://github.com/twbs/bootlint/blob/master/src/bootlint.js#L991

If there is anyway I could help with this though please let me know! If only the wiki's could accept pull requests :laughing:

**Comment by kkirsche** [comment]
# W015

## Detected what appears to be Bootstrap v4 or later. This version of Bootlint only supports Bootstrap v3.

Based on the version of Bootstrap's jQuery plugins or based on the apparent version number in the URL of a Bootstrap-related `<script>` or `<link rel="stylesheet">` tag, it appears that you are using version 4 of Bootstrap. Bootlint only supports version 3.

This check is heuristic and thus can have false positives or negatives depending on what URLs you're using for Bootstrap or for Bootstrap-related third-party libraries.

**Comment by kkirsche** [comment]
# W014

## Carousel controls and indicators should use `href` or `data-target` to reference an element with class `.carousel`.

For proper processing of click events within a carousel each anchor within `.carousel-control` should include either a `href` or `data-target` attribute with the ID of the corresponding image.

Wrong:
```html
<a class="right carousel-control" role="button" data-slide="next">
```

Right:
```html
<a class="right carousel-control" href="#carousel-example-generic" role="button" data-slide="next">
```

Also Right:
```html
<a class="right carousel-control" data-target="#carousel-example-generic" role="button" data-slide="next">
```




Hope these help some at least as a starting point :+1:

**Comment by zacechola** [comment]
These look good. I'll add them. Thanks!

**Comment by kkirsche** [comment]
Thanks @zacechola :smiley: Trying to help out wherever / however I can.

**Comment by zacechola** [comment]
Always appreciated!

**Comment by kkirsche** [comment]
Going to close this as you've added those :smiley:
